delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutes.scala

```diff
@@ -98,6 +102,24 @@ class BlazegraphViewsRoutes(
                       }
                     )
                   },
+                  // Query a blazegraph view
+                  (pathPrefix("sparql") & pathEndOrSingleSlash) {
+                    implicit val um: FromEntityUnmarshaller[String] =
```

Comment:
It isn't very readable to put it inside of this block. Why not having the `FromEntityUnmarshaller` on top of the class or on the companion object?

if you don't want it to be for every `String`, it is probably better to:

```scala
final case class SparqlResponse(value: String) extends AnyVal
```

And then have the Unmarshaller on the companion object

Reply:
This is `FromEntityUnmarshaller` for the request query, it's needed there to enforce correct content type on request. I agree that it can probably be moved somewhere else.